lib/datadog/tracing/contrib/propagation/sql_comment/mode.rb

```diff
@@ -20,6 +20,14 @@ def service?
             def full?
               mode == Ext::FULL
             end
+
+            def prepend?
```

Comment:
What is the purpose of having both `prepend?` and `append?`? Is this to satisfy an externally defined API? Because this PR only calls one of the two (`append?`).

Reply:
Yeah, just API symmetry. I'll remove it.